src/templates/partials/stunnel.hbs

```diff
@@ -0,0 +1,35 @@
+# {{output.header}}
+# {{{output.link}}}
+{{#if (minver "5.40" form.serverVersion)}}
+{{#if (minver "1.1.1" form.opensslVersion)}}
+sslVersionMin = TLSv1.3
+ciphersuites = {{{join output.cipherSuites ":"}}}
+{{else}}
+  {{#if (minver "1.1.0" form.opensslVersion)}}
+sslVersionMin = TLSv1.2
```

Comment:
It looks like in [Version 4.54, 2012.10.09, urgency: MEDIUM](https://www.stunnel.org/NEWS.html) TLSv1.2 is supported with OpenSSL 1.0.1
> New parameters to configure TLS v1.1/v1.2 with OpenSSL version 1.0.1 or higher (thx to Henrik Riomar).

Can you share where the mapping of TLSv1.2 and openssl 1.1.0 comes from?

Reply:
According to the documentation of the configuration file
sslVersionMin This option is only available when compiling with OpenSSL 1.1.0 or higher.
If compiling with a version of openssl lower than 1.1.0 should use an option like "options = NO_TLSv1".

Link:
https://www.stunnel.org/static/stunnel.html

Reply:
Got it, ya the docs say

> Use sslVersionMax or sslVersionMin option instead of disabling specific TLS protocol versions when compiled with OpenSSL 1.1.0 or later.